rust-toolchain.toml

```diff
@@ -1,5 +1,4 @@
 [toolchain]
-channel = "1.82.0"
-targets = ["wasm32-unknown-unknown"]
+channel = "1.83.0"
+targets = ["wasm32-unknown-unknown", "x86_64-unknown-linux-gnu"]
```

Comment:
Why do we want a more recent version than the IC repo? https://github.com/dfinity/ic/blob/master/rust-toolchain.toml